ArduCopter/GCS_Mavlink.h

```diff
@@ -23,6 +25,11 @@ class GCS_MAVLINK_Copter : public GCS_MAVLINK
     Compass *get_compass() const override;
     AP_ServoRelayEvents *get_servorelayevents() const override;
     AP_GPS *get_gps() const override;
+    MAV_RESULT handle_flight_termination(const mavlink_command_long_t &packet) override;
+
+#if ADVANCED_FAILSAFE == ENABLED
+    AP_AdvancedFailsafe *get_advanced_failsafe() const override;
```

Comment:
The GCS_MAVLink already has an implementation that returns nullptr. I'd still need the #if in there so I'm not sure what the advantage would be?